client/components/Navbar.jsx

```diff
@@ -55,7 +55,9 @@ export default class Navbar extends React.Component {
         <div className='header position-sticky'>
           <h2 className='header-lover-h2'>BOOK<span className='header-lover'>LOVER</span></h2>
           <div className='dropdown-list-holder'>
-            <a onClick={this.getCategories} className='dropdown' data-view='on'>NYT Best Sellers<span className='span-category'>{categoryToShow}</span></a>
+            <a href='#'
+            onClick={this.getCategories}
+            className='dropdown' data-view='on'>NYT Best Sellers<span className='span-category'>{categoryToShow}</span></a>
```

Comment:
I recommend getting the categories in `componentDidMount` instead of `onClick`

Reply:
<img width="937" alt="Screen Shot 2022-09-15 at 9 21 12 AM" src="https://user-images.githubusercontent.com/106217355/190456332-2b546f89-74ab-4521-a82a-ccdb7b55b93d.png">